vendor/tracing/zero/destinations.ml

```diff
@@ -32,7 +31,72 @@ let direct_file_destination ?(buffer_size = 4096 * 16) ~filename () =
   (module Dest : Writer_intf.Destination)
 ;;
 
-let file_destination ~filename () = direct_file_destination ~filename ()
+let compressed_file_destination ?(buffer_size = 64 * 1024) ~filename () =
+  let buf = Iobuf.create ~len:buffer_size in
+  let compressed_size = buffer_size * 2 in
```

Comment:
Why not `Zstandard.compression_output_size_bound`, instead of `compressed_size*2`? One example where `compressed_size*2` is not quite right is if the file is empty for some reason (the zstd'd file will not also be empty).

Reply:
Also, very minor side note: comments go above the line you're commenting on, not below.